econml/dr/_drlearner.py

```diff
@@ -420,11 +420,52 @@ def __init__(self, *,
                          mc_iters=mc_iters,
                          mc_agg=mc_agg,
                          discrete_treatment=True,
-                         treatment_featurizer=None,
+                         treatment_featurizer=None,  # treatment featurization not supported with discrete treatment
                          discrete_instrument=False,  # no instrument, so doesn't matter
                          categories=categories,
                          random_state=random_state)
 
+    def const_marginal_effect(self, X=None):
```

Comment:
Correct, was just overridden to change docs.

The corresponding inference method docstrings need not be updated because their description is general enough to not mention treatment featurization.